os/include/tinyara/sched.h

```diff
@@ -490,6 +490,14 @@ struct task_group_s {
 
 	struct group_shm_s tg_shm;	/* Task shared memory logic                 */
 #endif
+
+#if defined(CONFIG_PREFERENCE) && CONFIG_TASK_NAME_SIZE > 0
+	/* Preference **************************************************************** */
```

Comment:
can you change it like below:

/**********Preference******************************
* comment
* comment
*/

Reply:
Your opinion is right but I did to follow other comments in this file.
/* modules *****************/
/* descriptions
 *
 */